lib/crds/redirects.rb

```diff
@@ -42,16 +57,19 @@ def to_csv!(path = './redirects.csv', debug=true)
       rows.delete_at(n)
     end
 
-    rows.insert(n, *redirects)
+    flex_page_redirects_data = flex_page_redirects
```

Comment:
we need to add logic to also be able to pull in drafted flex page slugs but they should only be appended to the list if the environment is demo env.

for prod we only want to add published entries to redirects.rb